packages/dds/tree/src/shared-tree/treeCheckout.ts

```diff
@@ -728,6 +732,54 @@ export class TreeCheckout implements ITreeCheckoutFork {
 		return { age: revertAge };
 	}
 
+	private forkRevertible(
```

Comment:
@jenn-le 

Instead of creating an external function, I decided to make this a `private` function inside the `TreeCheckout` class since I could not access private methods and objects such as `revertRevertible()`, `disposeRevertible()` & `logger`.